database/migrations/2022_09_07_141854_rename_capabilities.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class RenameCapabilities extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		DB::beginTransaction();
+
+		// flip the locked value
+		$ids = DB::table('users')->where('is_locked', '=', true)->get('id')->pluck('id');
+		DB::table('users')->whereIn('id', $ids)->update(['is_locked' => 0]);
+		DB::table('users')->whereNotIn('id', $ids)->update(['is_locked' => 1]);
+		DB::commit();
```

Comment:
Why that difficult? Why not simply

```suggestion
		DB::table('users')->update(['is_locked' => DB::raw('NOT is_locked'));
```

I haven't tried it, so maybe it does not work with SQLite or MySQL, but it should work in ANSI SQL.